WcaOnRails/app/views/competitions/_index_table.html.erb

```diff
@@ -0,0 +1,13 @@
+<ul class="list-group">
+  <li class="list-group-item"><strong><%= title %></strong></li>
+  <% competitions.each_with_index do |competition, index| %>
+    <li class="list-group-item <%= competition.is_over? ? "past" : "not-past" %>">
+      <span class="date"><%= date_range(competition.start_date, competition.end_date, separator: '-', show_year: false) %>, <%= competition.year %></span>
+      <span class="competition-info">
+        <p class="competition-link"><%= link_to competition.name, competition_path(competition) %></p>
+        <p class="location"><strong><%= competition.country.name %></strong>, <%= competition.cityName %></p>
+        <%=md(competition.venue, target_blank: true) %>
```

Comment:
That method is inside a migration. Will you run it soon?
I'm guessing you mean I should use the method in the mean time, but where should it go?